watertap/unit_models/electrodialysis_0d.py

```diff
@@ -0,0 +1,938 @@
+    **EnergyBalanceType.energyPhase** - energy balances for each phase.}"""))
+    '''
+
+    CONFIG.declare(
+        "momentum_balance_type",
+        ConfigValue(
+            default=MomentumBalanceType.pressureTotal,
+            domain=In(MomentumBalanceType),
+            description="Momentum balance construction flag",
+            doc="""Indicates what type of momentum balance should be constructed,
+    **default** - MomentumBalanceType.pressureTotal.
+    **Valid values:** {
+    **MomentumBalanceType.none** - exclude momentum balances,
+    **MomentumBalanceType.pressureTotal** - single pressure balance for material,
+    **MomentumBalanceType.pressurePhase** - pressure balances for each phase,
+    **MomentumBalanceType.momentumTotal** - single momentum balance for material,
+    **MomentumBalanceType.momentumPhase** - momentum balances for each phase.}""",
+        ),
+    )
+
+    CONFIG.declare(
+        "property_package",
+        ConfigValue(
+            default=useDefault,
+            domain=is_physical_parameter_block,
+            description="Property package to use for control volume",
+            doc="""Property parameter object used to define property calculations,
+    **default** - useDefault.
+    **Valid values:** {
+    **useDefault** - use default package from parent model or flowsheet,
+    **PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+
+    CONFIG.declare(
+        "property_package_args",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property packages",
+            doc="""A ConfigBlock with arguments to be passed to a property block(s)
+    and used when constructing these,
+    **default** - None.
+    **Valid values:** {
+    see property package for documentation.}""",
+        ),
+    )
+
+    def build(self):
+        # build always starts by calling super().build()
+        # This triggers a lot of boilerplate in the background for you
+        super().build()
+        # this creates blank scaling factors, which are populated later
+        self.scaling_factor = Suffix(direction=Suffix.EXPORT)
+
+        # Next, get the base units of measurement from the property definition
+
+        # Create essential sets.
+        self.membrane_set = Set(initialize=["cem", "aem"])
+
+        # Create unit model parameters and vars
+        self.water_density = Param(
+            initialize=1000,
+            mutable=False,
+            units=pyunits.kg * pyunits.m**-3,
+            doc="density of water",
+        )
+        self.water_MW = Param(
+            initialize=18.015e-3,
+            mutable=False,
+            units=pyunits.kg * pyunits.mole**-1,
+            doc="molecular weight of water",
+        )
+        self.cell_pair_num = Var(
+            initialize=1,
+            domain=NonNegativeIntegers,
+            bounds=(1, Inf),
```

Comment:
```suggestion
            bounds=(1, 1000),
```
We should actually bound this variable to something reasonable.